pkg/util/netlink.go

```diff
@@ -101,14 +101,57 @@ func LinkDelete(link string) error {
 	return err
 }
 
-// Listen for events on a specific interface and callback if any. The interface
-// does not have to exist. Use the stop channel to stop listening.
+func isSuitableMacvtapParent(link netlink.Link) bool {
+	switch link.(type) {
+	case *netlink.Bond, *netlink.Device:
+		return true
+	}
+	return false
```

Comment:
We probably need to work on this a bit more. Like filtering out loopback devices, etc. Hopefully catch these on the tests.